packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -27,7 +26,14 @@ export const Breadcrumb = React.forwardRef<HTMLElement, HTMLAttributes<HTMLEleme
         );
       }
 
-      return <StyledBreadcrumbItem>{child}</StyledBreadcrumbItem>;
+      return (
+        <>
+          <StyledBreadcrumbItem>{child}</StyledBreadcrumbItem>
+          <StyledBreadcrumbItem style={{ display: 'flex' }} aria-hidden>
```

Comment:
Why does this one get an inline `css` designation?

Reply:
Good question! **I applied a `style` attribute to `StyledBreadcrumbItem` (which renders an `<li>`) so that the contents of the `<li>` can be centered**. In this particular case, the content is a chevron SVG icon. The HTML for this UI looks like:

```html
<ol> // flex container
  <li>one</li>
  <li> // make this a flex container to center chevron
    <svg>chevron</svg>
  </li>
  <li>two</li>
  <li>
    <svg>chevron</svg>
  </li>
  <li>three</li>
</ol>
```

Alternatively, I could have relied on the `<ol>` flex container, and I did not have to apply flex to each `<li>` that had a chevron. That would look like the following:

```html
<ol> // flex container
  <li>one</li>
  <svg>chevron</svg> // centered because it's a direct child of `<ol>` flex container
  </li>
  <li>two</li>
  <svg>chevron</svg> // centered because it's a direct child of `<ol>` flex container
  <li>three</li>
</ol>
```

I avoided the second alternative because the [HTML spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ol-element) recommends only `<li>` as the contents of an `<ol>`.

Reply:
If this requires custom styling, we should be applying this through styled-components. I think it could still live in this file though.

```tsx
const StyledCenteredBreadcrumbItem = styled(StyledBreadcrumbItem)`
  display: flex;
`;
```

Reply:
@rossmoody @austin94  I've cleaned this up, thank you.